src/main/scala/scorex/core/network/MaliciousBehaviorException.scala

```diff
@@ -0,0 +1,8 @@
+package scorex.core.network
+
+/**
+  * Custom exception to distinguish malicious behaviour of external peers from non-adversarial network issues
+  *
+  * @param message - exception message
```

Comment:
Doesn't sounds like valuable description. :) The description can contain some hints on what kind of information should be provided in the message. This will also help to understand where this exception should and shouldn't be used.

Reply:
There are no limits for the message semantics I guess.